src/renderer/coremods/contextMenu/index.tsx

```diff
@@ -0,0 +1,216 @@
+import type {
+  ContextItem,
+  GetContextItem,
+  RawContextItem,
+} from "../../../types/coremods/contextMenu";
+import { Logger } from "../../modules/logger";
+import { getByProps } from "../../modules/webpack";
+import { React } from "@common";
+
+const logger = Logger.api("ContextMenu");
+
+/**
+ * An enum for the navIds of context menus across discord
+ * @enum {string}
+ */
+export enum NavIds {
```

Comment:
Can we rename this to something more clear? Maybe `ContextMenuTypes` or something like that. Also I think it should be moved to the types folder.